ci-operator/step-registry/openshift/e2e/aws/builds/openshift-e2e-aws-builds-workflow.yaml

```diff
@@ -0,0 +1,13 @@
+workflow:
```

Comment:
Do we want to invent a new syntax for "copy this previous workflow but override" by re-using the logic we have for test cases in a singular `ci-operator` configuration?

Reply:
I think the complexity of the implementation (execution, validation, resolution, rehearsals, …) does not justify it for this case. The steps are already factored into chains, so there is only three lines of repeated content in a few files that are all in the same place and could be easily changed together.

If we found a more complex case that would give greater benefits, I'd be fine with implementing it then coming back and DRYing this.